components/apimgt/org.wso2.carbon.apimgt.core/src/main/java/org/wso2/carbon/apimgt/core/impl/APIPublisherImpl.java

```diff
@@ -580,8 +594,8 @@ public WorkflowResponse updateAPIStatus(String apiId, String status, Map<String,
                 apiBuilder.lifecycleState(currentState);
                 for (Map.Entry<String, Boolean> checkListItem : checkListItemMap.entrySet()) {
                     getApiLifecycleManager().checkListItemEvent(api.getLifecycleInstanceId
-                            (), api.getLifeCycleStatus(),
-                    checkListItem.getKey(), checkListItem.getValue());                   
+                                    (), api.getLifeCycleStatus(),
```

Comment:
Formatting issue getLifecycleInstanceId &  () should be in one line like getLifecycleInstanceId()